eas.json

```diff
@@ -1,16 +1,17 @@
 {
   "cli": {
-    "version": ">= 0.60.0"
+    "version": ">= 0.60.0",
+    "appVersionSource": "remote"
   },
   "build": {
-    "development": {
-      "developmentClient": true,
-      "distribution": "internal"
-    },
```

Comment:
do we really want to get rid of this build config? at some point it seems to me that `preview` should be the same as a production build, i.e. `developmentClient` is going to be `false` so that we know that preview users are getting as close to a production experience as possible.

Reply:
Yeah, I guess now that we're back to using Expo Go we don't really need the development client at all - this was a holdover change from the before times, I didn't want us to have to do two builds per commit (if we would have needed the dev client to do anything). 

Let me reinstate this and remove the dev client from `preview`